theano/gpuarray/opt.py

```diff
@@ -2182,6 +2247,11 @@ def local_gpu_svd(op, context_name, inputs, outputs):
 # to avoid a circular dependency problem with dnn
 from .dnn import (local_abstractconv_cudnn, local_abstractconv_gw_cudnn,
                   local_abstractconv_gi_cudnn)     # noqa: 402
+
+conv_metaopt = ConvMetaOptimizer()
+abstractconv_groupopt.register('conv_metaopt', conv_metaopt, 0, 'conv_meta')
```

Comment:
Interesting... in Theano 0.8, `LocalGroupDB` is a subclass of `SequenceDB`, which has [`def register(self, name, obj, position, *tags)`](https://github.com/Theano/Theano/blob/0.8.X/theano/gof/optdb.py#L311).
In Theano 0.9, `LocalGroupDB` overrides this method as [`def register(self, name, obj, *tags, **kwargs):`](https://github.com/Theano/Theano/blob/0.9.X/theano/gof/optdb.py#L417). While it still passes the positional arguments to `SequenceDB.register()` (including the position), it also overrides the `query` method in a way that only `position=...` keyword arguments to `register` are considered.
I don't know if the interface was broken on purpose, but yes, it means you will have to use `position=0` for it to work.